cloud/google/node_init.sh

```diff
@@ -6,4 +6,21 @@ sudo mkdir ${DATA_DIR}
 sudo /usr/share/google/safe_format_and_mount \
   -m "mkfs.ext4 -F" ${DATA_DEV} ${DATA_DIR}
 
-
+# Install google-fluentd which pushes application log files up into the Google
+# Cloud Logs Monitor.
+curl -sSO https://storage.googleapis.com/signals-agents/logging/google-fluentd-install.sh
+sha256sum ./google-fluentd-install.sh
```

Comment:
I assume this outputs the SHA256 of the script you just downloaded.
You don't do anything with the output and go on to do sudo things.
Seems risky.